.github/workflows/deploy-to-production.yml

```
@@ -0,0 +1,56 @@
+name: 🚀 Deploy To Production Workflow
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)